test/controllers/responsiblepeople/TimeAtAdditionalExtraAddressControllerSpec.scala

```diff
@@ -204,7 +204,9 @@ class TimeAtAdditionalExtraAddressControllerSpec extends AmlsSpec with MockitoSu
     }
   }
   "App must start" in {
-    val app = GuiceApplicationBuilder().build()
+    val app = GuiceApplicationBuilder()
```

Comment:
Did we say we were keeping these specs or removing them? Not sure if the keystoreConnector bind has something to do with it but It seems pointless keeping tests to see if the app can start.

Reply:
I've never heard about removing those specs, but in order to run this spec, bind is needed

Reply:
but I could :)

Reply:
done